### DISCUSSION TITLE
Application Libraries

### DISCUSSION THREAD

**Discussion by execsumo** [ClassicOldSong/Apollo#57] [Ideas]
A way to organize and group Applications would be very useful. 
For example, I'd love to separate kids' games from my games. Something like folders or groups would do the trick.

**Comment by ClassicOldSong** [comment]
I have plan about either removing the app list as it's totally pointless from the point of separating what apps you can access, or integrate the list with Playnite directly for better functionalities.

Still haven't decided which way to go, ideas are welcomed!

**Reply by execsumo** [reply]
Just downloaded and tried Playnite, and an integration with it would cover my use-case and probably many others (like not having to manually add games to the application list). It'll make the Apollo / Artemis experience more like a console, which IMO is great progress. 

You'll probably unlock a ton of potential (less technical?) users. With the progress you're making, the tools are becoming mainstream consumer friendly. Keep it up!

**Reply by booyakafoo** [reply]
> I have plan about either removing the app list as it's totally pointless from the point of separating what apps you can access, or integrate the list with Playnite directly for better functionalities.
> 
> Still haven't decided which way to go, ideas are welcomed!

Hello.  I know this is old and maybe something has already been implemented and I just don't see it.  When I launch and connect to my PC, I have the Playnite and Virtual Desktop applications.  Is it possible to just launch into the Playnite app without selecting it?  I have everything already setup there and it would be nice to remove the step to manually launch Playnite.  Thanks.
